build_docs.py

```diff
@@ -745,6 +745,9 @@ def build(self):
         blurb = self.venv / "bin" / "blurb"
 
         if self.includes_html:
+            # For opengraph previews (requires matplotlib)
+            sphinxopts.append("-t create-social-cards")
```

Comment:
And so this will be used later in `conf.py` to decide whether to create social cards.